renovate-x86/tests/X64.hs

```diff
@@ -79,10 +81,12 @@ mkTest :: C.HandleAllocator -> FilePath -> T.TestTree
 mkTest hdlAlloc fp = T.testCase fp $ withELF elfFilename testRewrite
   where
     elfFilename = replaceExtension fp "exe"
+    testRewrite :: E.SomeElf E.ElfHeaderInfo -> IO ()
     testRewrite elf = do
       Just expected <- readMaybe <$> readFile (fp <.> "expected")
-      let cfg = [(R.X86_64, R.SomeConfig NR.knownNat MBL.Elf64Repr (R64.config (R.AnalyzeOnly (analysis expected))))]
-      R.withElfConfig (E.Elf64 elf) cfg (testBlockRecovery hdlAlloc)
+      let cfg = [(R.X86_64
+                 , R.SomeConfig NR.knownNat MBL.Elf64Repr (R64.config (R.AnalyzeOnly (analysis expected))))]
```

Comment:
This looks like a formatting-only change. I like to keep formatting changes and functional changes in separate commits where possible. This makes it easier to review changes (the reviewer doesn't have to comb carefully through formatting-only changes to make sure they didn't miss something), and makes commits more focused on their intended changes.

This is just a nit/note for the future, and shouldn't hold up merging.

Reply:
Yes, I normally avoid that as well; I think this one just crept in accidentally.